### ISSUE TITLE
Emqx startup CPU usage

### ISSUE THREAD

**Issue by lewisdiamond** [emqx/emqx#2485]
#### Environment

- OS: Linux - Docker - Kubernetes
- Erlang/OTP: Latest emqx image

#### Description
erl_child_setup takes 100% CPU for a long time (tens of minutes to an hour) then settles.

```
  PID USER      PRI  NI  VIRT   RES   SHR S CPU% MEM%   TIME+  Command
  4536 ldiamond   20   0   760     8     0 R 100.  0.0  0:26.08 erl_child_setup 1073741816
```

Repost of https://github.com/emqx/emqx-chart/issues/4 since this may be a more appropriate place to report it.

**Comment by turtleDeng** [comment]
Already fixed in v3.2.3, please download the latest version.

**Comment by mythic41** [comment]
![image](https://user-images.githubusercontent.com/58205079/69811142-658db900-1228-11ea-8988-7a537b6e3f2b.png)
